test/sqllogictest/transform/threshold_elision.slt

```diff
@@ -479,29 +479,27 @@ WITH MUTUALLY RECURSIVE
 SELECT * FROM c1
 ----
 Explained Query:
-  Return // { non_negative: true }
-    Get l0 // { non_negative: true }
+  Return // { non_negative: false }
+    Get l0 // { non_negative: false }
```

Comment:
Looks like regressions.

Reply:
Not actually regressions. At least, non-regressed assessments of improved plans.